### ISSUE TITLE
With v6.0.18, Sidekiq doesn't run at all

### ISSUE THREAD

**Issue by blanchma** [mhenrixon/sidekiq-unique-jobs#471]
**Describe the bug**
The sidekiq process run but no jobs are processed. 

**Expected behavior**
A job is enqueued. A job is processed.

**Current behavior**
No job is processed with or without unique locks.

**Worker class**

```ruby
class SidekiqHealthJob
  include Logging
  include Sidekiq::Worker

  def perform
    UniqueJob.perform_async

    redis = Configuration.redis
    sidekiq_health = Configuration.redis.get "sidekiq_health"

    if sidekiq_health && 5.minute.ago.to_i < sidekiq_health.to_i
      true
    elsif sidekiq_health
      false
      Rollbar.error("Sidekiq Health Job run last time #{Time.at(sidekiq_health)}")
    else
      Rollbar.error("Sidekiq Health Job never ran")
    end
  end

  class UniqueJob
    include Logging
    include Sidekiq::Worker
    sidekiq_options(retry: 3, lock_expiration: 10, unique: :until_executing)

    def perform
      sidekiq_health = Configuration.redis.set "sidekiq_health", Time.now.to_i
    end
  end
end

```

**Additional context**
sidekiq (6.0.4)
      connection_pool (>= 2.2.2)
      rack (>= 2.0.0)
      rack-protection (>= 2.0.0)
      redis (>= 4.1.0)
    sidekiq-unique-jobs (6.0.18)
      concurrent-ruby (~> 1.0, >= 1.0.5)
      sidekiq (>= 4.0, < 7.0)
      thor (~> 0)

ruby 2.5.7

If a downgrade to 6.0.13 to work again.

**Comment by mhenrixon** [comment]
Seems related to https://github.com/mhenrixon/sidekiq-unique-jobs/pull/435

**Comment by mhenrixon** [comment]
If you change to `sidekiq_options(retry: 3, lock_expiration: 10, lock: :until_executing)` it should work properly @blanchma

**Comment by mhenrixon** [comment]
Fixed in v6.0.19 and I also yanked the broken gems
